website/cue/reference/components/sources.cue

```diff
@@ -166,9 +166,11 @@ components: sources: [Name=string]: {
 							type: string: {
 								default: "bytes"
 								enum: {
-									bytes:  "Events containing the byte frame as-is."
-									json:   "Events being parsed from a JSON string."
-									syslog: "Events being parsed from a Syslog message."
+									bytes:       "Events containing the byte frame as-is."
+									json:        "Events being parsed from a JSON string."
+									syslog:      "Events being parsed from a Syslog message."
+									native:      "Events being parsed from Vector's [native protobuf format](\(urls.native_proto_schema)) (EXPERIMENTAL)."
+									native_json: "Events being parsed from Vector's [native JSON format](\(urls.native_json_schema)) (EXPERIMENTAL)."
```

Comment:
Until we have docs for codecs, maybe it'd be good to link to the highlight you added?

Reply:
How do you think that would be best to include? Something like `(see highlight)` after `(EXPERIMENTAL)`, or maybe make `(EXPERIMENTAL)` itself the link?

Reply:
Mmm, I like making `EXPERIMENTAL` itself the link.